packages/manager/modules/pci/src/projects/project/sidebar/sidebar.less

```diff
@@ -0,0 +1,10 @@
+cloud-sidebar-project-list {
+  @import '~@ovh-ux/ui-kit/dist/less/tokens/_colors';
```

Comment:
Should this import be set a L1 of the file ?

Reply:
Would that not mean a global import? We actually need it only inside this class...